keygen/src/keygen.rs

```diff
@@ -532,11 +540,15 @@ fn do_main(matches: &ArgMatches<'_>) -> Result<(), Box<dyn error::Error>> {
                 exit(1);
             }
 
+            let num_threads =
+                value_t!(matches.value_of("num_threads"), usize).unwrap_or(num_cpus::get());
```

Comment:
With `.default_value()` and `.validator()` now in place, it's safe to use `value_t_or_exit!()` here since the value of `matches.value_of("num_threads")` is now fully sanitized by clap
```suggestion
                value_t_or_exit!(matches.value_of("num_threads"), usize);
```